transformation/src/main/scala/org/broadinstitute/monster/hca/HcaPipelineBuilder.scala

```diff
@@ -32,9 +33,14 @@ object HcaPipelineBuilder extends PipelineBuilder[Args] {
   // format is: metadata/{entity_type}/{entity_id}_{version}.json,
   // but filename returns just {entity_id}_{version}.json, so that is what we deal with.
   val metadataPattern: Regex = "([^_]+)_(.+).json".r
-  // format is: {dir_path}/{file_id}_{file_version}_{file_name},
+  // format is: data/{dir_path}/{file_id}_{file_version}_{file_name},
```

Comment:
This string comes from the list of filenames within the `data` directory so I don't think the `data/` part is included

Reply:
I don't think it would be included for the links pattern below either, but I could be wrong.

Reply:
Ah that makes sense - I only added that to match the metadata piece above.
I was trying to get them all to match what's in the design document but I can see now that it might be confusing for someone reading the code and trying to understand what the regex is doing. I'll go ahead and remove the two I added as well as the metadata one! Thanks!